Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -900,15 +934,15 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
     private func setExcludedRoutes(_ excludedRoutes: [IPAddressRange], completionHandler: ((Data?) -> Void)? = nil) {
         Task {
             self.excludedRoutes = excludedRoutes
-            try? await updateTunnelConfiguration(selectedServer: settings.selectedServer, reassert: false)
+            try? await updateTunnelConfiguration(reassert: false)
             completionHandler?(nil)
         }
     }
 
     private func setIncludedRoutes(_ includedRoutes: [IPAddressRange], completionHandler: ((Data?) -> Void)? = nil) {
         Task {
             self.includedRoutes = includedRoutes
-            try? await updateTunnelConfiguration(selectedServer: settings.selectedServer, reassert: false)
+            try? await updateTunnelConfiguration(reassert: false)
```

Comment:
(Sorry for repetition, but adding for clarity) Wouldn't removing the environment here potentially change the selected environment?